### PR TITLE
fix(BBD-880): Search page doesn't load on refresh

### PR DESCRIPTION
**Pull request by Filip-F**
https://github.com/groupby/flux-capacitor/pull/147
https://github.com/groupby/storefront-presets/pull/54
https://github.com/groupby/storefront-products/pull/30
https://github.com/groupby/storefront-paging/pull/18
https://github.com/groupby/storefront-breadcrumbs/pull/22
https://github.com/groupby/storefront-navigation/pull/34
https://github.com/groupby/storefront-did-you-mean/pull/17
https://github.com/groupby/storefront-related-queries/pull/17
https://github.com/groupby/storefront-record-count/pull/20

### CODE REVIEW

File: src/navigation/index.ts [hunk 1]
```diff
@@ -18,10 +18,12 @@ class Navigation {
   };
 
   init() {
+    this.updateFields();
```
Comment (Filip-F) [inline src/navigation/index.ts]: Do `this.select(Selectors.navigationsObject)` here, and don't change the original function.

File: test/unit/navigation.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ suite('Navigation', ({ expect, spy, itShouldBeConfigurable, itShouldHaveAlias })
   describe('init()', () => {
     it('should set initial fields', () => {
       navigation.flux = <any>{ on: () => null };
+      navigation.updateFields = spy();
```
Comment (Filip-F) [inline test/unit/navigation.ts]: Doesn't need to be a spy.

File: test/unit/navigation.ts [hunk 3]
```diff
@@ -35,26 +36,41 @@ suite('Navigation', ({ expect, spy, itShouldBeConfigurable, itShouldHaveAlias })
...
 
       navigation.init();
 
       expect(on).to.be.calledWith(Events.NAVIGATIONS_UPDATED, navigation.updateFields);
     });
+
+    it('should call updateFields', () => {
+      const on = spy();
+      const updateFields = navigation.updateFields = spy();
+      navigation.flux = <any>{ on };
```
Comment (Filip-F) [inline test/unit/navigation.ts]: Doesn't need to be a spy.

File: test/unit/navigation.ts [hunk 4]
```diff
@@ -27,34 +27,54 @@ suite('Navigation', ({ expect, spy, itShouldBeConfigurable, itShouldHaveAlias })
...
+      navigation.updateFields = () => null;
+      navigation.select = () => null;
       navigation.flux = <any>{ on };
 
       navigation.init();
 
       expect(on).to.be.calledWith(Events.NAVIGATIONS_UPDATED, navigation.updateFields);
     });
+
+    it('should call updateFields', () => {
```
Comment (Vic-Dev) [inline test/unit/navigation.ts]: Should check it's called w/ result of `select`

Comment (zfcb) [reply test/unit/navigation.ts]: Test has been changed 👍

**Pull request merged**